governance-exercises/VoteWizard_poll_exercise_04/Emergency Response Poll - Risk mitigation options - August 9, 2023.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Emergency Response Poll - Risk mitigation options - August 9, 2023
```

Comment:
A little vague, can probably make it a bit better, without causing too much alarm.